src/Unit/Algebra.hs

```diff
@@ -111,14 +80,14 @@ instance (Unit dimu u, Unit dimv v) => Unit (dimu :*: dimv) (u :*: v) where
 
 newtype Inv (u :: * -> *) a = Inv { getInv :: a }
   deriving (Conjugate, Epsilon, Enum, Eq, Foldable, Floating, Fractional, Functor, Integral, Num, Ord, Real, RealFloat, RealFrac, Scalar, Show, Storable, Traversable, GL.Type, Uniform)
-  deriving (Additive, Applicative, Metric, Monad) via Identity
+  deriving (Additive, Applicative, Metric, Monad) via I
```

Comment:
I wouldn’t have minded getting rid of `Inv` and using `:/:` in its stead, but it makes the `Mul` family much more complicated to have two different 2-combinators to worry about.